packages/libs/error-stack/tests/snapshots/doc/hook__debug_hook_provide.snap

```diff
@@ -1,7 +1,7 @@
 <b>invalid user input</b>
-<span style='color:#a00'>├</span><span style='color:#a00'>╴</span><span style='color:#a00'> </span><span style='color:#555'>src/hook.rs:49:14</span>
 <span style='color:#a00'>├</span><span style='color:#a00'>╴</span><span style='color:#a00'> </span>suggestion: try better next time!
 <span style='color:#a00'>├</span><span style='color:#a00'>╴</span><span style='color:#a00'> </span>error code: 420
+<span style='color:#a00'>├</span><span style='color:#a00'>╴</span><span style='color:#a00'> </span><span style='color:#555'>src/hook.rs:49:14</span>
```

Comment:
It's a bit unfortunate, that `Location` will be printed after all provided values, but as this also applies to backtraces and span traces this is fine I guess.

Reply:
Yea, this is all about tradeoffs, I mean we had a previous design where this would have been possible to reorder, but that one was just too confusing. I think this is fine tho, as it is still visually distinct.